### PR TITLE
fix: change heroicon to fontawesome

### PR DESCRIPTION
**Pull request by zalborzi**
Changes requested by @brunopacheco1 here: https://github.com/GenomicDataInfrastructure/gdi-userportal-frontend/pull/336

## Summary by Sourcery

This pull request updates the icon library used in the HomePage component from Heroicons to FontAwesome. This change addresses compatibility and consistency issues by replacing the ArrowTopRightOnSquareIcon with the FontAwesomeIcon component using the faArrowUpRightFromSquare icon.

- **Bug Fixes**:
    - Replaced Heroicons with FontAwesome icons for better compatibility and consistency.

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -7,7 +7,8 @@ import PageContainer from "@/components/PageContainer";
 import SearchBar from "@/components/Searchbar";
 import { useSearchParams } from "next/navigation";
 import bgBackground from "../public/bg-background.png";
-import { ArrowTopRightOnSquareIcon } from "@heroicons/react/20/solid";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faArrowUpRightFromSquare } from "@fortawesome/free-solid-svg-icons";
```
[the remark this answers is not in the document]
Comment (zalborzi) [reply src/app/page.tsx:11]: That's how it's been used in different pages.

**Comment by brunopacheco1** [comment]
I will bring the coments to this PR.

File: src/app/page.tsx [hunk 2]
```diff
@@ -7,7 +7,8 @@ import PageContainer from "@/components/PageContainer";
 import SearchBar from "@/components/Searchbar";
 import { useSearchParams } from "next/navigation";
 import bgBackground from "../public/bg-background.png";
```
Comment (brunopacheco1) [inline src/app/page.tsx:9]: important: `src/public/bg-background.png` is too big, please optime the size.

Comment (brunopacheco1) [inline src/app/page.tsx:9]: important: rename `bg-background.png` to `homepage-about-background.png`, for a cleaner and clearer name.

rename variable from `bgBackground` to `aboutBackground`.

File: src/app/page.tsx [hunk 3]
```diff
@@ -60,7 +61,11 @@ const HomePage = () => {
             className="text-primary flex items-center gap-1 transition hover:underline duration-1000"
```
Comment (brunopacheco1) [inline src/app/page.tsx:61]: important: fix height of `About the data portal`. The background is cut when the text is too short.

image

File: src/app/page.tsx (hunk 2 above)
Comment (zalborzi) [reply src/app/page.tsx]: it's already been

Comment (zalborzi) [reply src/app/page.tsx]: it's been already done

File: src/app/page.tsx (hunk 3 above)
Comment (zalborzi) [reply src/app/page.tsx:61]: min 300 is set.

**Comment by zalborzi** [comment]
Screenshot 2024-07-15 at 15 38 01

**Review by brunopacheco1 — APPROVED** [review]

**Pull request merged**